models/observations.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
@aman-tunerlabs - I think observations or assessment specific thing better be in samiksha only

Reply:
@aks30  I will make the changes